pod/perllocale.pod

```diff
@@ -55,11 +55,8 @@ currently no UTF-8 locales for EBCDIC platforms.
 
 (Unicode is also creating C<CLDR>, the "Common Locale Data Repository",
 L<http://cldr.unicode.org/> which includes more types of information than
-are available in the POSIX locale system.  At the time of this writing,
-there was no CPAN module that provides access to this XML-encoded data.
-However, it is possible to compute the POSIX locale data from them, and
-earlier CLDR versions had these already extracted for you as UTF-8 locales
-L<http://unicode.org/Public/cldr/2.0.1/>.)
+are available in the POSIX locale system.  There are various CPAN
+modules that provides access to this XML-encoded data.
```

Comment:
Could we mention a few such CPAN modules?

Reply:
done